e2e-commons/up.sh

```diff
@@ -5,8 +5,13 @@ set -e # exit when any command fails
 ./down.sh
 docker-compose build
 docker-compose up -d parity1 parity2 e2e
+export DOCKER_LOCALHOST="localhost"
 
 while [ "$1" != "" ]; do
+  if [ "$1" == "macos" ]; then
```

Comment:
When will this be used?

Reply:
This is used by me on macOS (or any other developer with macOS wanting to run these tests).

It's not used on CI because on linux this workaround is not needed.
The workaround is recommended by the [docs](https://docs.docker.com/docker-for-mac/networking/)

Reply:
so, you run it manually in pipeline like this 'e2e-commons/up.sh macos deploy native-to-erc', right?

Reply:
Exactly.